test/e2e/util/hypershift_framework.go

```diff
@@ -258,7 +260,23 @@ func (h *hypershiftTest) createHostedCluster(opts *core.CreateOptions, platform
 	opts, err = createClusterOpts(h.ctx, h.client, hc, opts)
 	g.Expect(err).NotTo(HaveOccurred(), "failed to generate platform specific cluster options")
 
+	// Dump the output from rendering the cluster objects for posterity
+	dumpDir := filepath.Join(artifactDir, artifactSubdirFor(h.T))
+	if err := os.MkdirAll(dumpDir, 0755); err != nil {
+		h.Errorf("failed to create dump directory: %v", err)
+	}
+
+	opts.Render = true
+	opts.RenderInto = filepath.Join(dumpDir, "manifests.yaml")
+	h.Logf("Dumping new cluster manifests to %s", opts.RenderInto)
+	if err := createCluster(h.ctx, hc, opts); err != nil {
```

Comment:
`createCluster()` also creates infra, so calling it just to render the manifest, will create extra unneeded infra which will also not be cleaned up.

Reply:
Interesting, I was hoping that it would be deterministic and re-use the infra the second time around.

Reply:
Updated